.github/workflows/github-action-scan.yml

```diff
@@ -3,27 +3,16 @@ name: Build SecHub GHA (scan)
 
 on:
   push:
-    branches:
-      - main
-      - master
-      - develop
-      - hotfix
+    paths:
+      - 'github-actions/**'
   pull_request:
-    # This workflow will run for all pull requests that target following branches
-    branches:
-      - main
-      - master
-      - develop
-      - hotfix
+    paths:
+      - 'github-actions/**'
```

Comment:
same here (if comment above is applicable)

Reply:
done